libraries/AP_NavEKF3/AP_NavEKF3_Logging.cpp

```diff
@@ -176,34 +172,22 @@ void NavEKF3_core::Log_Write_XKF5(uint64_t time_us) const
         return;
     }
 
-    // Write fifth EKF packet
-    float normInnov=0; // normalised innovation variance ratio for optical flow observations fused by the main nav filter
```

Comment:
lost all these helpful comments as part of this though..

Reply:
Yeah, I was feeling guilty about losing those, and was considering moving them back.  I'll do so.

They do repeat what's now in the Log message documentation (https://github.com/ardupilot/ardupilot/blob/master/libraries/AP_Logger/LogStructure.h#L2400) - but perhaps slightly different phrasing might not be a bad thing sometimes.

Reply:
Comments are back.